pkgs/top-level/php-packages.nix

```diff
@@ -655,7 +648,7 @@ in {
           buildInputs = [ net-snmp openssl ];
           configureFlags = [ "--with-snmp" ];
           # net-snmp doesn't build on darwin.
-          enable = (!stdenv.hostPlatform.isDarwin);
+          meta.broken = (!stdenv.hostPlatform.isDarwin);
```

Comment:
Isn't this wrong now? Before it was enabled on linux and now it is marked broken on linux.

Reply:
Yes, oops. But it turns out it builds fine on Darwin now. https://github.com/NixOS/nixpkgs/pull/353479

Reply:
I think pdo had the same mistake #353492